docs/integration.rst

```diff
@@ -927,6 +927,16 @@ DatastoreHook
     :members:
 
 
+Cloud Memorystore
+'''''''''''''''
+
+MemorystoreHook
+"""""""""""""
+
+.. autoclass:: airflow.contrib.hooks.gcp_memorystore_hook.MemorystoreHook
```

Comment:
The structure of `integration.rst` file has changed. Can you update your change?